File: benchmarks/json/parse_1mb.rb
require 'json'
data = JSON.generate(Array.new(20000) { |i| {id: i, name: "item_#{i}", value: i * 3.14} })
start = Process.clock_gettime(Process::CLOCK_MONOTONIC, :microsecond)
parsed = JSON.parse(data)
finish = Process.clock_gettime(Process::CLOCK_MONOTONIC, :microsecond)
puts(finish - start)

File: benchmarks/json/parse_1mb.js
const data = JSON.stringify(Array.from({length: 20000}, (_, i) => ({id: i, name: `item_${i}`, value: i * 3.14})));
const start = Date.now();
const parsed = JSON.parse(data);
console.log((Date.now() - start) * 1000);
